sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py

```diff
@@ -344,21 +334,17 @@ def import_certificate(
         contain the key as well as x509 certificates. This operation requires
         the certificates/import permission.
 
-        :param name: The name of the certificate.
-        :type name: str
-        :param certificate_bytes: Bytes of the ertificate object to import.
-        This certificate needs to contain the private key.
-        :type certificate_bytes: str
-        :param password: If the private key in base64EncodedCertificate is
-         encrypted, the password used for encryption.
-        :type password: str
+        :param str name: The name of the certificate.
+        :param bytes certificate_bytes: Bytes of the certificate object to import. This certificate
+            needs to contain the private key.
+        :param str password: If the private key in base64EncodedCertificate is
```

Comment:
What's `base64EncodedCertificate `?

Reply:
good catch, that was copied form the generated code and refers to the certificate bytes which used to be passed in by a base64 encoded string. Switching now, thanks!